scene/2d/animated_sprite_2d.h

```diff
@@ -43,7 +43,7 @@ class AnimatedSprite2D : public Node2D {
 	bool backwards = false;
 	StringName animation = "default";
 	int frame = 0;
-	float speed_scale = 1.0f;
+	float speed_scale = 1.0;
```

Comment:
In some places, I replaced `double` with `float`, since the value is stored in `float`. But I'm not entirely sure, maybe we should replace `float` with `double` here?

Reply:
It's unlikely anyone would assign ludicrous values to `speed_scale` because the animation ends up looking unrecognisable, so the extra accuracy of `double` would be unnecessary.

Reply:
I was a little concerned about this too, since Godot treats delta as a double, so delta should be used for internal calculations, but the data stored as SpriteFrames is always set from the editor, not converted from outside, so it is a double It does not seem necessary. Then I think this change is acceptable.